build.gradle

```diff
@@ -0,0 +1,103 @@
+plugins {
+    id 'java'
+    id 'eu.xenit.alfresco' version "0.2.0"
+    id 'eu.xenit.amp' version "0.2.0"
+    id "eu.xenit.docker-alfresco" version "4.0.2"
+}
+
+description = 'Remote-jodconverter for Alfresco'
+
+group = 'eu.xenit.transformers'
+version = '1.1.5-SNAPSHOT'
+
+sourceCompatibility = '1.8'
+
+repositories {
+    mavenLocal()
+}
+
+tasks.withType(JavaCompile) {
+    options.encoding = 'UTF-8'
+}
+
+ext.alfrescoVersion = "5.1.g"
+
+dependencies {
+    alfrescoProvided "org.alfresco:alfresco-repository:${alfrescoVersion}"
+
+    /* Alfresco with remote jod converter amp docker image */
+    alfrescoAmp files(project.amp)
+    baseAlfrescoWar "org.alfresco:alfresco:${alfrescoVersion}@war"
+    baseShareWar "org.alfresco:share:${alfrescoVersion}@war"    
+}
+
+
+dockerAlfresco {
+    baseImage = "hub.xenit.eu/alfresco-community:${alfrescoVersion}"
```

Comment:
This image is not available publicly

Reply:
You can base it on a skeleton image, then there is no alfresco war in there yet

Reply:
I added a comment that it will need to change with a public image, once it is available.

Skeleton images do not have any connector incorporated, that's why I used a community image + leanImage=false